A30-xds-v3.md

```diff
@@ -0,0 +1,255 @@
+A30: xDS v3 Support
+----
+* Author(s): Mark D. Roth (markdroth)
+* Approver: ejona86, dfawley
+* Status: In Review
+* Implemented in: 
+* Last updated: 2020-06-25
+* Discussion at: https://groups.google.com/g/grpc-io/c/YRVqRMUQwhM
+
+## Abstract
+
+The purpose of this proposal is to describe how gRPC will handle the
+transition from v2 to v3 of the xDS API and to document how gRPC will handle
+xDS versioning on an ongoing basis.
```

Comment:
In that case would it be better to rename the gRFC (and the filename) to something like "xDS versioning in gRPC" since that will be the long term purpose?

Reply:
I think the current title is probably going to seem more relevant to existing and near-term prospective users.